code/procs/jobprocs.dm

```diff
@@ -640,6 +640,8 @@
 			trinket = new/obj/item/reagent_containers/food/snacks/ingredient/egg/bee(src)
 	else if (src.traitHolder && src.traitHolder.hasTrait("smoker"))
 		trinket = new/obj/item/device/light/zippo(src)
+	else if (src.traitHolder && src.traitHolder.hasTrait("plasmatoid"))
```

Comment:
considering internals are Very Fuckin Important, may want to put this as the first thing in the list